modules/cluster-compare-templating.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="cluster-compare-templating_{context}"]
+= Configuring expected variation in a template
+
+You can handle variable content within a template by using Golang templating syntax. Using this syntax, you can configure validation logic that handles optional, required, and conditional content within the template.  
+
+[NOTE]
+====
+You can use the Golang templating syntax with custom and built-in functions for complex use cases. All Golang built-in functions are supported including the functions in the Sprig library. This implementation uses the Helm templating behavior and supports all custom functions that are used in Helm, such as `example: toYaml`.
```

Comment:
I think we should remove the part about Helm - we actually don't have all the Helm functions.  The first part about Sprig is correct. We have the functions in the Sprig library: https://masterminds.github.io/sprig/
```suggestion
You can use the Golang templating syntax with custom and built-in functions for complex use cases. All Golang built-in functions are supported including the functions in the Sprig library.
```

Reply:
Updated